src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPLINPACK.json

```diff
@@ -6,30 +6,38 @@
     "SupportedOperatingSystems": "Ubuntu"
```

Comment:
Do we support all Ubuntu or just Ubuntu22?

Reply:
Without performance libraries it is supported on all Ubuntu. But with performance library, only on Ubuntu 22. I have included this in the documentation.

Reply:
Then let's mention that in profile. you can put Ubuntu22 in the SupportedOperatingSystems